composites/Plugin/Shared/components/KeywordInput.js

```diff
@@ -1,73 +1,89 @@
+// External dependencies.
 import React from "react";
 import styled from "styled-components";
 import PropTypes from "prop-types";
+import uniqueId from "lodash/uniqueId";
+import { __ } from "@wordpress/i18n";
 
+// Internal dependencies.
 import colors from "../../../../style-guide/colors.json";
 
-let KeywordField = styled.input`
-	margin-right: 0.5em;
-	border-color: ${ props => props.borderColor };
+const errorColor = colors.$color_red;
+
+const KeywordInputContainer = styled.div`
+	display: flex;
+	flex-direction: column;
+	margin: 1em 0;
 `;
 
-const ErrorText = styled.div`
+const KeywordFieldLabel = styled.label`
 	font-size: 1em;
-	color: ${ colors.$color_red };
-	margin: 1em 0;
-	min-height: 1.8em;
+	font-weight: bold;
+	margin-bottom: 0.5em;
 `;
 
-const ErrorMessage = "Are you trying to use multiple keywords? You should add them separately below.";
+const KeywordField = styled.input`
+	border-color: ${ colors.$color_grey_light };
```

Comment:
this border color (`#f1f1f1`) seems a bit too light to me to have a good contrast, and also different from other input border colors. /Cc @hedgefield

Reply:
Good catch, it should be the same as the other input borders (given those have enough contrast)